pkg/cache/lrucache.go

```diff
@@ -59,6 +60,19 @@ func NewGitRepoLRUCache(dir string, minFreeGbs uint64) (*GitRepoLRUCache, error)
 		return nil, fmt.Errorf("error checking provided cache directory: %s", err.Error())
 	}
 
+	stats := &syscall.Statfs_t{}
+
+	err = syscall.Statfs(dir, stats)
+	if err != nil {
+		return nil, fmt.Errorf("error fetching stats for cache directory: %s", err.Error())	
+	} 
+
+	freeSpace := stats.Bavail * stats.Bfree
+	
+	if freeSpace <= minFreeGbs {
```

Comment:
`freeSpace` as written will only be available space in _bytes_. We'll need to do some at runtime math to make sure the actual byte value > miniumum free gbs:

See the `tryEvict` function for an example:

 https://github.com/open-sauced/pizza/blob/51ed9ca9798ad3f9c39fc02a368fc1f525949737/pkg/cache/lrucache.go#L193-L194